iam/api/QuickStartV2Test/QuickStartTest.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Iam.v1;
+using Google.Apis.Iam.v1.Data;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+public class QuickStartTest : IDisposable
+{
+    private readonly string projectId;
+    private ServiceAccount serviceAccount;
+    private IamService iamService;
+
+    public QuickStartNewTest()
+    {
+        // Check for _projectId and throw exception if empty
+        projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        if(projectId == null)

```

Comment:
NIT: I don't think that you need to throw an exception (I can't think of any other test where we do throw if the environment variable isn't set).

Reply:
Yes, we do throw in several samples when the env var is not set. And it's very helpfull because you inmediately know what you need to fix (instead of getting an error from client libs or APIs).

Reply:
Fair enough. I'll start adding those checks to my unit tests.